README.md

```diff
@@ -1,6 +1,49 @@
 # cnp-flux-config
 Weave Flux config for AKS clusters
 
+##Folder Structure
```

Comment:
this isn't rendering right atm
https://github.com/hmcts/cnp-flux-config/blob/d190d8d75646f2594eaa75ced4d6d2bba0b24909/README.md
```suggestion
## Folder Structure
```